src/main/java/io/github/thebusybiscuit/slimefun4/core/services/holograms/HologramsService.java

```diff
@@ -245,30 +245,20 @@ private Hologram getAsHologram(@Nonnull BlockPosition position, @Nonnull Entity
      * 
      * @param loc
      *            The {@link Location}
-     * @param consumer
-     *            The callback to run
      */
-    private void updateHologram(@Nonnull Location loc, @Nonnull Consumer<Hologram> consumer) {
+    @Nullable
+    private Hologram updateHologram(@Nonnull Location loc) {
```

Comment:
I definitely don't agree with the changes here, this turns update hologram essentially into a wrapper of getHologram and removes the functionality defined in the javadocs

Reply:
Hmm, you are totally right. So I think we can safely remove method `updateHologram` and replace its usages with `getHologram(loc, true)` or create a shorthand for this like `getHologramOrCreate`. What do you think about it?